### PR TITLE
Added Alrighty Snippets

### PR DESCRIPTION
**Pull request by r3nya**
Hey!

I want to add my snippets for JS (React.js).

Thanks! 👍

### CODE REVIEW

File: repository/a.json [hunk 1]
```diff
@@ -416,6 +416,20 @@
 			]
 		},
 		{
+			"name": "Alrighty Snippets",
+			"details": "https://github.com/alrighty/alrighty-snippets-sublime",
+			"issues": "https://github.com/alrighty/alrighty-snippets-sublime/issues",
```
Comment (wbond) [inline repository/a.json]: This is the default value, so you should remove it from the JSON

File: repository/a.json [hunk 2]
```diff
@@ -416,6 +416,20 @@
...
+			"readme": "https://raw.githubusercontent.com/alrighty/alrighty-snippets-sublime/master/readme.md",
```
Comment (wbond) [inline repository/a.json]: This is the default value, so you should remove it from the JSON

File: repository/a.json [hunk 3]
```diff
@@ -416,6 +416,20 @@
...
+			"labels": ["snippets", "react", "jsx", "es6"],
+			"author": "r3nya",
+			"releases": [
+				{
+					"sublime_text": "*",
+					"branch": "master"
```
Comment (wbond) [inline repository/a.json]: Please see https://packagecontrol.io/docs/submitting_a_package#Step_4

**Comment by FichteFoll** [comment]
@r3nya ping

**Comment by r3nya** [comment]
@FichteFoll sorry for my delay. I will fix it today. :)

**Comment by r3nya** [comment]
Hey @wbond @FichteFoll!
Please review my changes.

**Comment by FichteFoll** [comment]
What are the differences to other [react snippets packages](https://packagecontrol.io/search/react%20snippets)?

**Comment by r3nya** [comment]
@FichteFoll so … good question. I made [this snippets for Atom](https://atom.io/packages/alrighty-snippets) and my mates and me use it because another snippets has different code style.

**Comment by FichteFoll** [comment]
I suggest you include a short paragraph into your readme about what makes your package special and why you believe the style you chose is better (or what guidelines/references you follow).

**Pull request merged**